### ISSUE TITLE
Third deployment setup: Distributed Event Recording and Queuing, Centralized Processing

### ISSUE THREAD

**Issue by ahabari** [padams/Open-Web-Analytics#21]
Hi, I am not sure if this is the right place to post this. I hope someone will be able to elaborate about this setup. 

I read the following sections on the wiki: Deployment,Event processing,Event queuing and the CLI sections. But I am still missing some parts. 

What I understood:
1. A load balancer distributes request to several servers.
2. Each server queues the events on a local file
3. This is the part I am missing
4. The different event files are now on the admin server.
5. Use the cli util on admin server to process event into DB 

So basically my question is how do we get the local events to  the the admin server.
Just copy the files? Which files? archived/unprocessed/events.txt ?

Thanks
